### PR TITLE
MINOR: KAFKA-3989_follow_up_PR: update script to run from kafka root eg ./jm…

### PR DESCRIPTION
**Pull request by bbejeck**
…h-benchmarks/jmh.sh

### CODE REVIEW

**Comment by bbejeck** [comment]
@ijuma - script updated  `./jmh-benchmarks/jmh.sh` works now

File: jmh-benchmarks/jmh.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ jmh_project_name="jmh-benchmarks"
 
 if [ ${base_dir} == "." ]; then
      gradlew_dir=".."
-elif [ ${base_dir} == ${jmh_project_name} ]; then
+elif [ ${base_dir} == "./${jmh_project_name}" ]; then
```
Comment (ijuma) [inline jmh-benchmarks/jmh.sh]: Does `jmh-benchmarks/jmh.sh` still work?

**Comment by bbejeck** [comment]
@ijuma - my apologies for the oversight.  Confirmed the following all work with this update.

1. ./jmh.sh <args>
2. ./jmh-benchmarks/jmh.sh <args>
3.   jmh-benchmarks/jmh.sh <args>

**Comment by guozhangwang** [comment]
LGTM. Merged to trunk and cherry-picked to 0.11.0.

**Pull request closed without merging**